website/app/styles/markdown/other-elements.scss

```diff
@@ -1,60 +1,59 @@
-.doc-markdown-content {
-
-  // IMAGES
-
-  > img,
-  > p img,
-  > li img,
-  > th img,
-  > td img {
-    box-sizing: content-box;
-    max-width: 100%;
-    background-color: #fff; // TODO
-
-    &[align="right"] {
-      padding-left: 20px;
-    }
-
-    &[align="left"] {
-      padding-right: 20px;
-    }
-  }
+// LINKS
 
-  // CODE SNIPPETS
-  // TODO do we need this or in reality we will not use them directly?
+.doc-markdown-a {
+  color: teal;
+}
 
-  code { // for the typographic styles see "typography"
-    margin: 0;
-    padding: 4px 8px;
-    color: white; // TODO
-    background-color: black; // TODO
-    border-radius: 3px;
-  }
+// IMAGES
 
-  pre {
-    word-wrap: normal;
+.doc-markdown-img {
+  box-sizing: content-box;
+  max-width: 100%;
+  background-color: #fff; // TODO
+
+  &[align="right"] {
+    padding-left: 20px;
+  }
```

Comment:
Are these styles just temporary ones for testing? If not, you may want to avoid using the `align` attribute on `img` tags as it's deprecated. https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/align

Nit: If you are trying to just add spacing between a floated `img` and surrounding text then I would normally use `margin` in preference to `padding`.

Reply:
I have removed for now the styling in case of align (we'll figure out later if we need it).